projects/packages/videopress/src/client/block-editor/blocks/video/components/tracks-control/index.tsx

```diff
@@ -0,0 +1,114 @@
+/**
+ * External dependencies
+ */
+import { NavigableMenu, MenuItem, MenuGroup, ToolbarButton, Dropdown } from '@wordpress/components';
+import { useCallback } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { upload } from '@wordpress/icons';
+/**
+ * Internal dependencies
+ */
+import { TrackProps, VideoControlProps } from '../../types';
+import { captionIcon } from '../icons';
+import './style.scss';
+import { TrackListProps } from './types';
+import type React from 'react';
+
+/**
+ * Track Item component.
+ *
+ * @param {VideoControlProps} props - Component props.
+ * @returns {React.ReactElement}      TracksControl block control
+ */
+function TrackItem( { label, kind }: TrackProps ): React.ReactElement {
+	return (
+		<div className="videopress-block__track-item">
+			{ label }
+			<span className="videopress-block__track-item-kind"> ({ kind })</span>
+		</div>
+	);
+}
+
+/**
+ * Track List React component.
+ *
+ * @param {VideoControlProps} props - Component props.
+ * @returns {React.ReactElement}      TracksControl block control
+ */
+function TrackList( { tracks }: TrackListProps ): React.ReactElement {
+	if ( ! tracks?.length ) {
+		return (
+			<div className="videopress-block-tracks-control__track_list">
+				{ __(
+					'Tracks can be subtitles, captions, chapters, or descriptions. They help make your content more accessible to a wider range of users.',
+					'jetpack-videopress-pkg'
+				) }
+			</div>
+		);
+	}
+
+	return (
+		<MenuGroup
+			className="videopress-block-tracks-control__track_list"
+			label={ __( 'Text tracks', 'jetpack-videopress-pkg' ) }
+		>
+			{ tracks.map( ( track: TrackProps, index ) => {
+				return (
+					<TrackItem
+						key={ index }
```

Comment:
The index as key is React's default behavior, ideally something involving the track data would be better.